docs/developer.md

```diff
@@ -150,20 +144,37 @@ $ cd $GOPATH/src/github.com/openebs/maya
  make bin
  ```
 
- #### Test
+### Test your changes
 
  ```sh
  cd $GOPATH/src/github.com/openebs/maya
 
  # Run every unit test
  make test
-```
-#### Keep your branch in sync
+ ```
+
+### Keep your branch in sync
+
+[Rebasing](https://git-scm.com/docs/git-rebase) is very import to keep your branch in sync with the changes being made by others and to avoid huge merge conflicts while raising your Pull Requests. You will always have to rebase before raising the PR. 
 
 ```sh
 # While on your myfeature branch (see above)
 git fetch upstream
 git rebase upstream/master
 ```
 
-#### After making changes to your local branch and running the `test`, create a pull request:
+While you rebase your changes, you will be required to resolve any conflicts that might arise and build and test your changes using the above steps. 
```

Comment:
you will be required to resolve = you must resolve

Reply:
done